src/Fantomas/Context.fs

```diff
@@ -84,6 +84,10 @@ module WriterModel =
                 { m with
                       Lines = "" :: m.Lines
                       Column = 0 }
+            | Write s when s.Equals("(*") ->
```

Comment:
I'm pretty sure this will match more than just code comments.
```fsharp
let v = //
    (*) 4 8
```

This does not look safe at all. @jindraivanek any thoughts?

Reply:
Agree.
Proper place to change BlockComment behavior is at https://github.com/fsprojects/fantomas/blob/795854da9df94c5c4e20bf1271697b4eb205a089/src/Fantomas/Context.fs#L1108-L1113

I would try to use `atIndentLevel true 0` to output something from the start of the line.

Reply:
@jindraivanek but dont you think fantomas should put the comment where it originally was in the source? not just at the start of the line always (see convo in the other hunk below)

Reply:
@jindraivanek @knocte I was trying to modify the suggested block of code, however any changes only affects the generation of "live code", but it does not affect the comments generation. I can see that the comments are saved in ContentBefore/ContentAfter properties. These properties save the comments as raw string (I mean without "Range" that specifies start line and start column). Therefore, there is no way to control where the comment should be located. Is it rigth?

Reply:
@HumbertoCortes Right, to achieve this, I think `BlockComment` will need to be extended with original Column position.

Reply:
Indeed, I was also thinking in that direction.
You will need to detect the indentation in 
https://github.com/fsprojects/fantomas/blob/c1a4fe61fc4089eada7b38ef8f872ab28f8b914c/src/Fantomas/TokenParser.fs#L655-L703

And extend `BlockComment` to carry that the extra information.
Perhaps you can get away with just adding the missing spaces to the text that was found in the token.
Let me know if you get what I mean.